### DISCUSSION TITLE
ProHD just stopped working...

### DISCUSSION THREAD

**Discussion by pfuller88** [GerryDazoo/Slinger#576] [General]
Hi all.  My ProHD has been working fine since last November with slinger.  All of a sudden it stopped.  I went to check the box and noticed the Ethernet light was not on and two of the lights in the sling logo were solid.  The power light was on.  When I rebooted, all most immediately the same LED lights mentioned above come on, no ethernet light.   I tried the reset many times, same thing?  I'm thinking the circuit board may be poohed?   Any thoughts?  Any one have this happen?

**Comment by sp222222** [comment]
mine acted up about 2 weeks ago. it seemed like the hard reset button would not work even when pressed in. I could not ping it. the red light stayed on. I changed two power supplies and same thing.... I now have a different PROHD in use and am using a KASA plug instead of the reset button to use for reboots.

**Reply by pfuller88** [reply]
You're a life saver, I never thought of the power cable.  I switched the cable and it booted up properly.  I had to reset it, run through the unbrick steps, it is now working again.  FYI, I too use a wifi power switch to reboot.

Thanks sp222222

**Reply by htwonpoke** [reply]
I’m running into this same problem. Won’t respond to reset or anything. Where did you find another power cable?

**Reply by pfuller88** [reply]
I have several slingboxes so cables were lying around
